.travis.sh

```diff
@@ -43,20 +43,9 @@ pgdg_repositories() {
 }
 
 postgresql_install() {
-	if [ -z "${PGVERSION:-}" ]; then
-		PGVERSION="$( psql -d postgres -XAtc "select regexp_replace(current_setting('server_version'), '[.][0-9]+$', '')" )"
-	else
-		sudo service postgresql stop
-		xargs sudo apt-get -y --purge remove <<-packages
-			libpq-dev
-			libpq5
-			postgresql
-			postgresql-client-common
-			postgresql-common
-		packages
-		sudo rm -rf /var/lib/postgresql
-	fi
-
+        if [ -z "${PGVERSION:-}" ]; then
+                PGVERSION="$( psql -d postgres -XAtc "select regexp_replace(current_setting('server_version'), '[.][0-9]+$', '')" )"
```

Comment:
There's a simpler way to do it:

```
psql -At -c "select current_setting('server_version_num')::int / 10000"
```

How to get there:

```
pgloader# select version();
┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                   version                                                    │
├──────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ PostgreSQL 10.10 on x86_64-apple-darwin14.5.0, compiled by Apple LLVM version 7.0.0 (clang-700.1.76), 64-bit │
└──────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
(1 row)

pgloader# select name, setting from pg_settings where name ~ 'version';
┌────────────────────┬─────────┐
│        name        │ setting │
├────────────────────┼─────────┤
│ server_version     │ 10.10   │
│ server_version_num │ 100010  │
└────────────────────┴─────────┘
(2 rows)

pgloader# select current_setting('server_version_num')::int / 10000;
┌──────────┐
│ ?column? │
├──────────┤
│       10 │
└──────────┘
(1 row)
```

Reply:
OK, will do that before merging. Thanks for the comment!

Reply:
@dimitri Question - now that the buildmatrix only has jobs that explicitly specify `PGVERSION`, is there any sense in keeping this whole codeblock? It's effectively dead code now that it doesn't run in Travis.

```bash
        if [ -z "${PGVERSION:-}" ]; then
                PGVERSION="$( psql -At -c "select current_setting('server_version_num')::int / 10000" )"
        fi
```

Reply:
Oh dead code... then we should remove it I think, thanks for this maintenance efforts, much appreciated!

Reply:
OK - the current version of my PR has that code removed.

I am ready to squash&merge!

Reply:
If you have more time to spend on our tests I would suggest to have a look at the src/regress tooling in the Lisp code and see how to support database sources rather than single-table sources, so that we could run SQLite and MySQL migration tests in Travis too...

That's for another PR, of course, and would be great to have that. Another of the things I have in the back of my mind and never got around to...

Reply:
Created an issue for that.

Should I merge this one up?

Reply:
Yeah looks like it's ready!